apps/admin-gui/src/app/users/pages/user-detail-page/user-settings/user-roles/user-roles.component.ts

```diff
@@ -43,22 +43,38 @@ export class UserRolesComponent implements OnInit {
   }
 
   getData(): void {
+    let loadingTypes = 3;
     this.outerLoading = true;
+
```

Comment:
While this works a better solution (without rxjs magic) would be to nest these calls. Also stop the loading if some of the requests fail.